### PR TITLE
Remove unneeded GO_PACKAGES_EXPANDED

### PR DESCRIPTION
**Pull request by tnozicka**
/assign @sttts

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -10,4 +10,5 @@ $(call add-bindata,monitoring,./pkg/operator/staticpod/controller/monitoring/man
 $(call add-bindata,installer,./pkg/operator/staticpod/controller/installer/manifests/...,bindata,bindata,./pkg/operator/staticpod/controller/installer/bindata/bindata.go)
 $(call add-bindata,staticpod,./pkg/operator/staticpod/controller/prune/manifests/...,bindata,bindata,./pkg/operator/staticpod/controller/prune/bindata/bindata.go)
 
-GO_BUILD_PACKAGES :=./pkg/...
+GO_PACKAGES :=./pkg/...
+GO_BUILD_PACKAGES :=$(GO_PACKAGES)
```
Comment (sttts) [inline Makefile]: what's goal with GO_BUILD_PACKAGES?

Comment (sttts) [reply Makefile]: put that into a makefile comment?

Comment (tnozicka) [reply Makefile]: done

**Comment by tnozicka** [comment]
comment addressed, applying the label to unblock multibinary build PR

**Pull request merged**